Envio questionario

geovanahck opened the pull request.

MateusMiotto committed:
WIP Filtro tela EnvioQuestionario

MateusMiotto committed:
Commit

Implementado controller, service e repository questionário para a função de buscar superiores

MateusMiotto committed:
Commit WIP Method Post envio Questionario

MateusMiotto committed:
Merge branch 'listaQuestionario' of https://github.com/geovanahck/gestao-de-questionario into envioQuestionario

MateusMiotto committed:
WIP: Salvar envio de questionario

MateusMiotto committed:
Commit

Adicionado salvar questionario enviado e
gerar codigo para questionario

MateusMiotto committed:
Commit

Tela de Listagem de questionarios enviados
feita parcial

MateusMiotto committed:
Retoques

MateusMiotto committed:
Merge branches 'main' and 'envioQuestionario' of https://github.com/geovanahck/gestao-de-questionario into envioQuestionario

MateusMiotto committed:
Commit

Adicionado hora e datas do front end da tela de envio de questionario, salvar no banco e exibir na tela de questionarios enviados

PauloVitorSchio commented on src/main/resources/templates/envioQuestionario.html:
```diff
@@ -5,167 +5,425 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script rel="javascript" type="text/javascript" th:src="@{/scripts/envioQuestionario.js}"></script>
```
Coloca essa parte no final, última coisa antes do < /body>

MateusMiotto committed:
Commit

ajustes detalhes das telas e do javascript

MateusMiotto committed:
Merge branch 'main' of https://github.com/geovanahck/gestao-de-questionario into envioQuestionario

MateusMiotto committed:
Merge branch 'main' of https://github.com/geovanahck/gestao-de-questionario into envioQuestionario

MateusMiotto committed:
Commit

Commit

MateusMiotto committed:
Merge branch 'main' of https://github.com/geovanahck/gestao-de-questionario into envioQuestionario

MateusMiotto committed:
Update login.html

Commit merge

MateusMiotto committed:
Commit

MateusMiotto committed:
Commit

Melhorado front end e adicionado validação do campo date para tela de administradores

MateusMiotto committed:
Update application-local.properties
